integration/src/main/java/org/assimbly/integration/impl/CamelIntegration.java

```diff
@@ -1627,18 +1631,19 @@ public String getFlowStats(String flowId, boolean fullStats, boolean includeStep
 
 		flow.put("id",flowId);
 		flow.put("total",totalMessages);
-		flow.put("completed",completedMessages);
-		flow.put("failed",failedMessages);
-		flow.put("pending",pendingMessages);
+		flow.put("completedExchanges",completedMessages);
```

Comment:
I'm trying to avoid the term "exchange". It's a concept of Camel, but not of Assimbly. Exchange is a difficult concept as it has a lifecycle and patterns. I want to simplify it to just speak of "messages". So I deliberately removed Exchange everywhere.

Reply:
What do you think?

Reply:
Is this because Dovetail uses it. I think this concept is too difficult for Dovetail users.

Reply:
ok, I can change it.
I need to change this also on Dovetail, frontend and backend.

Reply:
done 👍🏻